lib/api/controller/document.js

```diff
@@ -744,6 +741,25 @@ class DocumentController extends NativeController {
       successes: response.items
     };
   }
+
+  _extractUpdatedFields (document, path = null, paths = []) {
```

Comment:
Why would this function define so many other possibilities and usages (given those extra params) if we don't use them :thinking: ?